io.github.wazzaps.Fingerpaint.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.wazzaps.Fingerpaint
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: "/app/bin/fingerpaint"
+finish-args:
+  # GUI Permissions
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--device=dri"
+
+  # Access to udev for finding raw touchpad device
+  - "--device=all"
+  - "--filesystem=/sys:ro"
+  - "--filesystem=/run/udev:ro"
+
+  # Access to dconf/gsettings for disabling touchpad under GNOME/Wayland
+  - "--filesystem=xdg-run/dconf:ro"
+  - "--filesystem=~/.config/dconf:ro"
+  - "--talk-name=ca.desrt.dconf"
+  - "--env=DCONF_USER_CONFIG_DIR=~/.config/dconf"
+modules:
+  - generated-poetry-sources.json
+  - name: fingerpaint
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py bdist_wheel && pip3 install --no-deps --no-index --prefix=/app dist/*.whl
+      - install -D flatpak/$FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/appdata/$FLATPAK_ID.appdata.xml
+      - install -D flatpak/$FLATPAK_ID.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -D snap/gui/fingerpaint.svg $FLATPAK_DEST/share/icons/$FLATPAK_ID.svg
+    sources:
+      - type: git
+        tag: "v1.4.1"
+        url: "https://github.com/wazzaps/fingerpaint"
+        commit: "9d1eac5b87ce9c4c4034eae742bf12ae457cbf6d"
+
+  # Taken from: https://archlinux.org/packages/extra/x86_64/xorg-xinput/
+  - name: xinput
+    buildsystem: simple
+    build-commands:
+      - zstd -d < xorg-xinput-1.6.4-1-x86_64.pkg.tar.zst | tar x usr/bin/xinput && mv usr/bin/xinput /app/bin/
+    sources:
+      - type: file
+        path: "externals/xorg-xinput-1.6.4-1-x86_64.pkg.tar.zst"
```

Comment:
so what happen when we build this on aarch64?

Reply:
Added a secondary source for aarch64